platform/linuxbsd/godot_linuxbsd.cpp

```diff
@@ -41,6 +41,18 @@
 #include <sys/resource.h>
 #endif
 
+// For export templates, add a section; the exporter will patch it to enclose
+// the data appended to the executable (bundled PCK)
```

Comment:
I didn't point it out initially because there was the same issue in the Windows code, but if we're doing style fixes there, let's also handle this one:
```suggestion
// For export templates, add a section; the exporter will patch it to enclose
// the data appended to the executable (bundled PCK).
```